lib/galaxy/tools/parameters/basic.py

```diff
@@ -163,9 +163,6 @@ def value_to_basic( self, value, app ):
         return self.to_string( value, app )
 
     def value_from_basic( self, value, app, ignore_errors=False ):
-        # HACK: Some things don't deal with unicode well, psycopg problem?
```

Comment:
It is old enough that we can assume library versions are fairly different. I'd say just drop the hack for now.